product_docs/docs/pgd/5/consistency/eager.mdx

```diff
@@ -50,15 +50,33 @@ In case of an origin node failure, the remaining nodes eventually (after at leas
 
 ## Effects of Eager Replication in general
 
-#### Increased commit latency
+#### Increased abort rate
 
-Adding a synchronization step means more communication between the nodes, resulting in more latency at commit time. Eager All-Node Replication adds roughly two network roundtrips (to the furthest peer node in the worst case). Logical standby nodes and nodes still in the process of joining or catching up aren't included but eventually receive changes.
+With single-node Postgres, or even with PGD in its default asynchronous
+replication mode, errors at `COMMIT` time are rare. The added synchronization
+step The added synchronization step due to the use of a commit scope using eager
```

Comment:
Duplicate _"The added synchronization step "_ here.

Reply:
Fixed - Thanks!